llama_index/embeddings/dashscope.py

```diff
@@ -155,34 +155,44 @@ class DashScopeEmbedding(MultiModalEmbedding):
 
                 - DashScopeTextEmbeddingModels.TEXT_EMBEDDING_V1
                 - DashScopeTextEmbeddingModels.TEXT_EMBEDDING_V2
+        text_type (str): The input type, ['query', 'document'],
+            For asymmetric tasks such as retrieval, in order to achieve better
+            retrieval results, it is recommended to distinguish between query
+            text (query) and base text (document) types, clustering Symmetric
+            tasks such as classification and classification do not need to
+            be specially specified, and the system default
+            value "document" can be used.
         api_key (str): The DashScope api key.
     """
 
     _api_key: Optional[str] = PrivateAttr()
+    _text_type: Optional[str] = PrivateAttr()
 
     def __init__(
         self,
         model_name: str = DashScopeTextEmbeddingModels.TEXT_EMBEDDING_V2,
+        text_type: str = "document",
```

Comment:
could you make a Enum for this text_type? so that users can know which options to select